apps/cde-ui/src/app/components/screen-size-notice/screen-size-notice.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="header">
```

Comment:
Use the directives from material when writing this component https://material.angular.io/components/dialog/api#MatDialogActions